vms/volunteer/tests/test_volunteerReport.py

```diff
@@ -65,7 +65,7 @@ def verify_shift_details(self, hours):
         :param total_shifts: Total number of shifts as filled in form.
         :param hours: Total number of hours as filled in form.
         """
-        total_no_of_hours = self.report_page.get_shift_summary().split(' ')[-1].strip('\n')
+        total_no_of_hours = self.report_page.get_report_hours()
```

Comment:
remove that previous method from pom object too, otherwise we'd have unused code.